Files/Interacts/Interaction.cs

```diff
@@ -656,7 +656,7 @@ public async void DeleteItem_Click(object sender, RoutedEventArgs e)
             }
             catch (IOException)
             {
-                if (await DialogDisplayHelper.ShowDialog(ResourceController.GetTranslation("FileInUseDeleteDialog.Title"), ResourceController.GetTranslation("FileInUseDeleteDialog.Text"), ResourceController.GetTranslation("FileInUseDeleteDialog.PrimaryButtonText"), ResourceController.GetTranslation("FileInUseDeleteDialog.SecondaryButtonText")))
+                if (await DialogDisplayHelper.ShowDialog(ResourceController.GetTranslation("FileInUseDeleteDialog/Title"), ResourceController.GetTranslation("FileInUseDeleteDialog/Text"), ResourceController.GetTranslation("FileInUseDeleteDialog/PrimaryButtonText"), ResourceController.GetTranslation("FileInUseDeleteDialog/SecondaryButtonText")))
```

Comment:
You are doing the same changes as in your another [PR](https://github.com/files-community/files-uwp/pull/1137/files#diff-0d4ba4bdc6e2e429e16d3f061cb662a9). You have to choose which one you want to leave.